server/sonar-server/src/main/java/org/sonar/server/authentication/BasicAuthenticator.java

```diff
@@ -76,6 +76,14 @@ public BasicAuthenticator(DbClient dbClient, CredentialsAuthenticator credential
     return new String[] {login, password};
   }
 
+  private static String getDecodedBasicAuth(String basicAuthEncoded) {
+    try {
+      return new String(BASE64_DECODER.decode(basicAuthEncoded.getBytes(Charsets.UTF_8)), Charsets.UTF_8);
+    } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)